index.html

```diff
@@ -11003,6 +11086,76 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
+		        <p>Authors SHOULD ensure that the type of text separation specified matches the visual rendering.</p>
+		        <p>The <pref>aria-textseparation</pref> <a>property</a> is an enumerated type. Any value not recognized in the list of allowed <span>values</span> SHOULD be treated by <a>assistive technologies</a> as if the value <code>inherit</code> had been provided. For future expansion, if the attribute is not present or its value is an empty string or <code>undefined</code>, the default value of <code>inherit</code> applies.</p>
+			</div>
+			<table class="property-features">
+				<caption>Characteristics:</caption>
+				<thead>
+					<tr>
+						<th scope="col">Characteristic</th>
+						<th scope="col">Value</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<th class="property-related-head" scope="row">Related Concepts:</th>
+						<td class="property-related">
+							<ul>
+								<li>String concatenation, CSS rendering</li>
+							</ul>
+						</td>
+					</tr>
+					<tr>
+						<th class="property-applicability-head" scope="row">Used in Roles:</th>
+						<td class="property-applicability">Placeholder</td>
+					</tr>
+					<tr>
+						<th class="property-descendants-head" scope="row">Inherits into Roles:</th>
+						<td class="property-descendants">Placeholder</td>
+					</tr>
+					<tr>
+						<th class="property-value-head" scope="row">Value:</th>
+						<td class="property-value"><a href="#valuetype_token">token</a></td>
+					</tr>
+				</tbody>
+			</table>
+			<table class="value-descriptions">
+				<caption>Values:</caption>
+				<thead>
+					<tr>
+						<th scope="col">Value</th>
+						<th scope="col">Description</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<th class="value-name" scope="row"><strong class="default">inherit (default)</strong></th>
+						<td class="value-description">Indicates that the element's text is separated from neighboring element text according to the element's display style.</td>
+					</tr>
+					<tr>
+						<th class="value-name" scope="row">line</th>
+						<td class="value-description">Indicates that the element's text is separated from neighboring element text by a line break.</td>
+					</tr>
+					<tr>
+						<th class="value-name" scope="row">none</th>
+						<td class="value-description">Indicates that the element's text is not separated from neighboring element text; it is rendered as a continuous whole, without any delineation.</td>
+					</tr>
+					<tr>
+						<th class="value-name" scope="row">paragraph</th>
+						<td class="value-description">Indicates that the element's text is separated from neighboring element text by a paragraph break.</td>
+					</tr>
+					<tr>
+						<th class="value-name" scope="row">space</th>
+						<td class="value-description">Indicates that the element's text is separated from neighboring element text by a space.</td>
```

Comment:
We still need leading and traling space options in cases where overriding inheritted behavior (see "1 cat" example in my earlier comment).

Reply:
We are now using the 2-token approach (first token is for leading, 2nd token is for trailing), which solves this.